Makefile

```diff
@@ -0,0 +1,3 @@
+docker-build:
+	docker build $(OPTFLAGS) -t rtradetech/ens-app:latest .
+	docker image push rtradetech/ens-app:latest
```

Comment:
Please add new line

Reply:
done as of https://github.com/ensdomains/ens-app/pull/581/commits/65467e692ff39838d8687ca3f4a3fd89f05a39e5